maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/ValidationServiceTest.java

```diff
@@ -293,62 +290,125 @@ void givenValidInputAndWithNewWorkReasonForUser_whenIsUserActionValidIsInvoked_t
     }
 
     @Test
     void givenValidInputWithOnlyNewWorkReason_whenIsUserActionValidIsInvoked_thenOKResponseIsReturned() throws CrimeValidationException {
-        UserValidationDTO userValidationDTO = TestModelDataBuilder.getUserValidationDTO();
-        userValidationDTO.setAction(null);
-        userValidationDTO.setSessionId(null);
+        UserActionDTO userActionDTO = TestModelDataBuilder.getUserActionDTO();
+        userActionDTO.setAction(null);
+        userActionDTO.setSessionId(null);
         UserSummaryDTO userSummaryDTO = TestModelDataBuilder.getUserSummaryDTO();
-        when(maatCourtDataService.getUserSummary(any())).thenReturn(userSummaryDTO);
-        Boolean isUserActionValid =
-                validationService.isUserActionValid(userValidationDTO);
+
+        Boolean isUserActionValid = validationService.isUserActionValid(userActionDTO, userSummaryDTO);
         assertTrue(isUserActionValid);
     }
 
     @Test
     void givenValidInputWithOnlySessionId_whenIsUserActionValidIsInvoked_thenOKResponseIsReturned() throws CrimeValidationException {
-        UserValidationDTO userValidationDTO = TestModelDataBuilder.getUserValidationDTOWithReservation();
-        userValidationDTO.setAction(null);
-        userValidationDTO.setNewWorkReason(null);
+        UserActionDTO userActionDTO = TestModelDataBuilder.getUserActionDTOWithReservation();
+        userActionDTO.setAction(null);
+        userActionDTO.setNewWorkReason(null);
 
         UserSummaryDTO userSummaryDTO = TestModelDataBuilder.getUserSummaryDTO();
         userSummaryDTO.getReservationsDTO().setUserSession(TEST_USER_SESSION);
-        when(maatCourtDataService.getUserSummary(any())).thenReturn(userSummaryDTO);
 
-        assertThat(validationService.isUserActionValid(userValidationDTO)).isTrue();
+        assertThat(validationService.isUserActionValid(userActionDTO, userSummaryDTO)).isTrue();
     }
 
     @Test
     void givenValidInputWithNoReservation_whenIsUserActionValidIsInvoked_thenOKResponseIsReturned() throws CrimeValidationException {
-        UserValidationDTO userValidationDTO = TestModelDataBuilder.getUserValidationDTO();
-        userValidationDTO.setAction(null);
-        userValidationDTO.setNewWorkReason(null);
+        UserActionDTO userActionDTO = TestModelDataBuilder.getUserActionDTO();
+        userActionDTO.setAction(null);
+        userActionDTO.setNewWorkReason(null);
 
         UserSummaryDTO userSummaryDTO = TestModelDataBuilder.getUserSummaryDTO();
         userSummaryDTO.setReservationsDTO(null);
-        when(maatCourtDataService.getUserSummary(any())).thenReturn(userSummaryDTO);
 
-        Boolean isUserActionValid =
-                validationService.isUserActionValid(userValidationDTO);
+        Boolean isUserActionValid = validationService.isUserActionValid(userActionDTO, userSummaryDTO);
         assertTrue(isUserActionValid);
     }
+
+    @Test
+    void isUserAuthorisedToEditField_returnsFalse_whenUserHasNoPermissions() {
```

Comment:
For the sake of consistency it would be good if we could stick to the `given_when_then` naming pattern.

Reply:
Sure thing 👍

Reply:
Maybe this is a topic for discussion we can all have thinking about it. I have always tended to prefer the style above because it puts the method under test at the beginning of the method name, which makes it easy to scan through and see / group your tests and locate tests. With the `given_when_then` it's certainly harder to achieve this.

Reply:
Done

Reply:
Sound like a good idea, unfortunately, we probably have hundreds of tests using the existing naming convention across several services, so not the easiest thing to change. That said, when we begin work on the next service, maybe this is one of the things we can incorporate - happy to have that conversation as a team.